guest-owner-proxy/pre-attestation.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+message BundleRequest {
+	bytes PlatformPublicKey = 1;
+	int32 Policy = 2;
```

Comment:
The policy value should originate from the guest owner, see here:

> The firmware maintains a guest policy provided by the guest owner. This policy is enforced by the 
> firmware and restricts what configuration and operational commands can be performed on this 
> guest by the hypervisor. The policy also requires a minimum firmware level. 
> The guest policy is provided to firmware during guest launch. The policy is then bound to the 
> guest and cannot be changed throughout the lifetime of the guest. The policy is also transmitted 
> during snapshot and migration flows and enforced on the destination platform. 
https://www.amd.com/system/files/TechDocs/55766_SEV-KM_API_Specification.pdf

Reply:
This follows the model of the launch secret endpoint where the fields are sent by the host but checked against the keyset configuration (and used to make the measurement). The policy is a bit less flexible than the firmware version and some of the other fields which the host could reasonably change (with guest owner permission), so maybe the GOP could directly select the policy. I'm not really sure which was is better. As is the policy is probably yet another thing to go in an annotation.

It's worth noting that the BundleRequest endpoint requires the policy because the policy is part of the launch bundle, but the policy can't be verified at this point because we haven't yet requested a particular keyset. The policy is required again for the LaunchSecret endpoint and the policy is verified there.